app/assets/javascripts/kuroko2/narrow_job_definitions.js

```diff
@@ -1,9 +1,17 @@
 jQuery(function ($) {
+  var link_to_table_row = function(){
+    $('tr[data-href]').css("cursor", "pointer").click(function(e){
```

Comment:
I don't like this kind of pseudo-links.
It cannot handle non-standard clicks such as opening context menu (right-click) and opening the link in new tab (middle-click).